larsim/PhotonPropagation/PhotonVisibilityService.h

```diff
@@ -298,6 +300,27 @@ namespace phot {
 
     MappedFunctions_t doGetTimingTF1(geo::Point_t const& p) const;
 
+    void findVoxelSuggestion(std::string& logString,
+                             float tpcMin,
+                             float tpcMax,
+                             float cryoMin,
+                             float cryoMax,
+                             int& nVoxels,
+                             float& voxelMin,
+                             float& voxelMax,
+                             float voxelSizeGoal = 10.0) const;
```

Comment:
Suggest removing the default value of 10.0 as it's not used anywhere (i.e. an explicit `voxelSizeGoal` argument is always specified when invoking `findVoxelSuggestion(...)`.

Reply:
Fixed.